lib/model/Manufacturer.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
+
+const manufacturers = require(path.join(__dirname, '..', '..', 'fixtures', 'manufacturers'));
```

Comment:
It may be considerable adding a static function that reloads this module for the case that the manufacturers change. As we designed the whole model to be capable of reloading with new data, it also makes sense here.